code/modules/projectiles/guns/ds13/contactbeam.dm

```diff
@@ -17,7 +17,7 @@
 	icon_state = "contact"
 	item_state = "contact"
 	wielded_item_state = "contact-wielded"
-	w_class = ITEM_SIZE_HUGE
+	w_class = ITEM_SIZE_BULKY
```

Comment:
why change this? all long guns should be huge to prevent them fitting in backpacks

Reply:
Mistype, I got them mixed at some point and changed it accidentally.